apps/storybook.namekit.io/stories/Namekit/Tooltip.stories.tsx

```diff
@@ -27,10 +18,6 @@ const meta: Meta<typeof Tooltip> = {
       control: { type: "number" },
     },
   },
-  args: {
```

Comment:
Removed these `args` because I've learned some new best practices with Storybook.

One key guideline is to focus on making sure you get really clean code examples on the "Docs" page:

![CleanShot 2024-09-18 at 23 36 36](https://github.com/user-attachments/assets/da022f15-3c63-4775-99fb-e4b05369118a)

If we use `args` here then it doesn't properly auto generate the code for cases where you want to show example code where an optional param is not defined. Ex: `maxTooltipWidth` in the screenshot above.